malfahad/fast-food-fast-react
# #164376310 Users should see past orders

#### What does this PR do?
- Introduces functionality to place and view orders

#### Description the task to be implemented
- users place orders
- admins see past orders
- users see past orders

#### How can this be manually tested?
- Fetch this branch by running `git fetch origin ft-view-orders-164376310`
- Checkout to this branch by running ` git checkout ft-view-orders-164376310`
- Install dependencies by running `npm install`
- Start the local server by running `npm start`
- navigate to the application and log in.
- use the menu to place an order
- you should see it in the order history

[#164376310](https://www.pivotaltracker.com/story/show/164376310)
